nms/app/fbcnms-projects/magmalte/app/views/equipment/GatewayDetailMain.js

```diff
@@ -99,8 +100,11 @@ export function GatewayDetail({
   enbInfo: {[string]: EnodebInfo},
 }) {
   const classes = useStyles();
-  const [tabPos, setTabPos] = React.useState(0);
   const {relativePath, relativeUrl, match} = useRouter();
+  const tabPos = findIndex(
+    ['overview', 'event', 'log', 'alert', 'config'],
+    route => location.pathname.startsWith(match.url + '/' + route),
+  );
```

Comment:
is there a way to share this code so that we are not repeating it?

also, is the list of items (i.e. `'overview', 'event', ...`) appearing anywhere else? would be good to store it all in one place

Reply:
Sure. I can do that.